UI/RemoteISOScreen.cpp

```diff
@@ -161,6 +162,10 @@ static void ExecuteServer() {
 		http->RegisterHandler(pair.first.c_str(), handler);
 	}
 
+	// TODO: This isn't an ideal place to put this... should separate.
+	// But kinda nice to only have it listen on one port?
```

Comment:
Yeah, I think this is fine really.

Reply:
Cool.  Thinking of moving it to Core/WebServer.cpp or something, and sending flags to it to restart with remote iso / debugger enabled.

-[Unknown]